prowler/providers/aws/services/elbv2/elbv2_service.py

```diff
@@ -50,6 +53,111 @@ def __describe_load_balancers__(self, regional_client):
             logger.error(
                 f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
+    
+    def __describe_target_groups__(self, regional_client):
+        logger.info("ELBv2 - Describing target groups...")
+        try:
+            for lb in self.loadbalancersv2:
+                try:
+                    if (
+                        lb.scheme == "internet-facing"
+                        and lb.type == "application"
+                        and len(lb.security_groups) > 0
+                    ):
```

Comment:
Why are you only getting in count public ELBs?

Reply:
fixed